tests/integration/env.bats

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bats
+# shellcheck disable=SC2016
+# This disables the check for shell variables inside single quotes
+# We do that all the time in this file, as we are testing env vars.
+
+load helpers
+
+function setup() {
+	setup_busybox
+}
+
+function teardown() {
+	teardown_bundle
+}
+
+@test "non-empty HOME env is used" {
+	update_config ' .process.env += ["HOME=/override"]'
+	update_config ' .process.args += ["-c", "echo $HOME"]'
```

Comment:
nit: can merge into single `update_config`

Reply:
Yeap, but I find it kind of painful when you need to modify it. IMHO this is more readable.

Do you think it is ok to keep it like this?

Reply:
Fine with me.